seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -147,6 +147,14 @@ bnb,babyswap,1,2022-11-22,0x0e6d9777fc533bc403df3ebe1b82b5399870f1301e45c90427b9
 bnb,babyswap,1,2022-11-30,0x279dfbce106e55f05a20331ed088e1fd2e1ff68c05ab089cca473f3dbbaef9fe,283,0x55d398326f99059ff775485246999027b3197955,1654.599686417120624489,0x53e562b9b7e5e94b81f10e96ee70ad06df3d2657,60085.854
 avalanche_c,trader_joe,1,2022-11-08,0x1f90838d916d2edf881ba53d8abfe5b557ff2a8d23d7ed692cc0687a71a9cd53,15,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,342510510,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,21741961515422881640
 avalanche_c,trader_joe,1,2022-11-09,0x82cd2992be090d073b82406e3f3882c43eeb4ac7e7ce26839290c6e028b2ac3b,130,0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e,30093631,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,2354558311510002477
+bnb,DODO,1,2021-09-15,0x51c65cb8860ca591782f72c4bbc661d234b9c9753733f5ab179122ee4abd10e3,568,0x55d398326f99059ff775485246999027b3197955,81.815,0xe9e7cea3dedca5984780bafc599bd69add087d56,81.827894594942210162
+bnb,DODO,1,2022-10-27,0x8a3127f41564e245ba378d77225ba902b9f827ab75c2d42933c9327055a8750b,69,0xe9e7cea3dedca5984780bafc599bd69add087d56,814.565954699970528680,0x55d398326f99059ff775485246999027b3197955,814.429100795211353004
```

Comment:
Since these are the only 2 seeds for `bnb, DODO, 1` trades.
These two seeds are the potential unmatched seeds in the [test failure](https://github.com/duneanalytics/spellbook/actions/runs/3690767818/jobs/6248120387#step:12:123).
[This tx 0x51c65cb8860ca591782f72c4bbc661d234b9c9753733f5ab179122ee4abd10e3](https://bscscan.com/tx/0x51c65cb8860ca591782f72c4bbc661d234b9c9753733f5ab179122ee4abd10e3) seems like a 0x:Exchange trade.

Reply:
Right, let me include other seeds and omit these ones to see if there is a change

Reply:
@Hosuke but would it be right to say that Dodo was used as a router and thus it qualfies to be a trade done on Dodo?

Reply:
The key point is there are seed records do not match the query result in your model.